pyartcd/pyartcd/pipelines/prepare_release_konflux.py

```
@@ -692,6 +701,84 @@ async def finalize_shipment(self, shipment_data: tuple):
+                return
+            if self.dry_run:
+                self.logger.info(
+                    "[DRY-RUN] Would have resolved advisory placeholders in advisory %s: %s",
+                    advisory_num,
+                    list(updates),
+                )
+                return
+            advisory.update(**updates)
+            advisory.commit()
+            self.logger.info("Resolved advisory placeholders in advisory %s: %s", advisory_num, list(updates))
+        except Exception as ex:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)